tests/test_examples.py

```diff
@@ -520,6 +520,26 @@ def test(self):
                 env_variables["PT_HPU_LAZY_MODE"] = "0"
                 env_variables["PT_ENABLE_INT64_SUPPORT"] = "1"
 
+            if self.EXAMPLE_NAME == "run_glue":
+                if model_name == "bert-large-uncased-whole-word-masking":
```

Comment:
no need to add 'llama_guard' model? We have both in README.

Reply:
which test ? i didnt see it ?

Reply:
https://github.com/huggingface/optimum-habana/blob/main/tests/utils.py#L79